src-electron/validation/validation.js

```diff
@@ -315,11 +291,26 @@ async function getBoundsInteger(attribute, typeSize, isSigned) {
   return {
     min: attribute.min
       ? await getIntegerFromAttribute(attribute.min, typeSize, isSigned)
```

Comment:
Not a big fan of this. Assign this to a variable and then check on that. Is more readable.

Reply:
We would need to check attribute.min first and then call getIntegerFromAttribute. Ideally we don't want to call getTypeBound if not required, since, validation is done at every keyboard entry.

Reply:
How would you suggest breaking this up?